packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -46,6 +47,11 @@
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test"
   },
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json",
+    "sfdx.debug.provideInitialConfigurations"
```

Comment:
Is there a reason why you need the "sfdx.debug.provideInitialConfiguration" as an activation event?

Reply:
Nope, think it came from copying another package.json, will fix